editor/plugins/script_editor_plugin.cpp

```diff
@@ -422,6 +422,8 @@ ScriptEditor *ScriptEditor::script_editor = nullptr;
 String ScriptEditor::_get_debug_tooltip(const String &p_text, Node *_se) {
 	String val = EditorDebuggerNode::get_singleton()->get_var_value(p_text);
 	if (!val.is_empty()) {
+		if (val.size() > 200)
+			val = val.substr(0, 200) + " [...] overflowed!";
```

Comment:
```suggestion
		if (val.size() > 200) {
			val = val.substr(0, 200) + " [...] overflowed!";
		}
```
Always use brackets

Reply:
Thanks!

Reply:
Make sure to copy the suggestion exactly